src/CSharp/CodeCracker/Style/PropertyPrivateSetAnalyzer.cs

```diff
@@ -39,7 +39,7 @@ private static void AnalyzeClass(SyntaxNodeAnalysisContext context)
             var invocationExpression = (PropertyDeclarationSyntax)context.Node;
             var semanticModel = context.SemanticModel;
 
-            if (invocationExpression.AccessorList.Accessors.Count == 1) return;
+            if (invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1) return;
```

Comment:
You're right. Reading code on the phone and reason about it doesn't seem to be a good idea. :smile:

But I'll try again.

.    invocationExpression.AccessorList?.Accessors.Count.GetValueOrDefault(1) == 1